### DISCUSSION TITLE
Authentication using laravel and Quasar

### DISCUSSION THREAD

**Discussion by cyberkid042** [quasarframework/quasar#11515] [Ideas / Proposals]
Hey guys,

I was wondering if there is any auth helper like in NUXTJS for laravel, etc. If there is none, can this act as a proposal/idea?

Thanks.

**Comment by metalsadman** [comment]
https://forum.quasar-framework.org/topic/6469/example-of-combining-laravel-8-and-quasar-superpowers-in-separate-repos?_=1638342216483

**Reply by cyberkid042** [reply]
Thanks for the help. I will give it a try and see if it works.

But out of curiosity, will quasar be doing same thing as Nuxt Auth in the future? And also adding route guards like Nuxt's middleware.

Thanks

**Reply by metalsadman** [reply]
no plans for that afaik.

**Comment by metalsadman** [comment]
check out @ldiebold new yt vid about auth https://youtu.be/CFa9RisAMi8

**Reply by ldiebold** [reply]
Working on it right now as you posted this! Thanks 😊

Just added supabase yesterday 🚀

**Comment by ldiebold** [comment]
Oh, just noticed you're looking for Laravel auth.
Feel free to copy paste some code from here!

https://github.com/ldiebold/sanctum-composables

**Wouldn't recommend using it as a dependency**, as it's in alpha and volatile right now!

Literally this morning I was working on implementing password reset and password update. The only two features missing before we can do a beta release :)
